libpod/runtime_pod_infra_linux.go

```diff
@@ -189,7 +189,12 @@ func (r *Runtime) makeInfraContainer(ctx context.Context, p *Pod, imgName, rawIm
 		g.AddLinuxSysctl(sysctlKey, sysctlVal)
 	}
 
-	containerName := p.ID()[:IDTruncLength] + "-infra"
+	containerName := p.config.InfraContainer.InfraName
+	if containerName == "" {
+		containerName = p.ID()[:IDTruncLength] + "-infra"
+	}
+	logrus.Debugf("Infra container name argument: %s", p.config.InfraContainer.InfraName)
+	logrus.Debugf("Infra container name %s", containerName)
```

Comment:
Upgrade this one to an Infof please

Reply:
Ops... I forgot to remove these lines. Do you want to keep it as `Infof` ?

Reply:
Oh... never mind. I missed the previous comment.